src/main.rs

```diff
@@ -64,6 +64,50 @@ fn main() -> Result<()> {
         .map(PathBuf::from)
         .expect("This should not happen, there's a default value for sigstore-cache-dir");
 
+    let policy_evaluation_limit = if matches.contains_id("disable-timeout-protection") {
+        None
+    } else {
+        match matches
+            .get_one::<String>("policy-timeout")
+            .expect("policy-timeout should always be set")
+            .parse::<u64>()
+        {
+            Ok(v) => Some(v),
+            Err(e) => {
+                fatal_error(format!(
+                    "'policy-timeout' value cannot be converted to unsigned int: {}",
+                    e
+                ));
+                unreachable!()
+            }
+        }
+    };
+
+    // Run in daemon mode if specified by the user
+    if matches.contains_id("daemon") {
```

Comment:
Is the daemonization of the process required just because of the tests? Or is there another reason?

Reply:
Yes, that's only needed by the e2e tests (so far). This made super easy to start policy-server from venom